Add instructions for Terragrunt folder (#9)
cds-snc/terraform-demo: terragrunt/README.md

> you can run it by doing the following:

Reviewer: Much appreciated for this!  I've just added a few tweaks:
```suggestion
# Instructions
You can run it by doing the following:
```

```
cd terragrunt/env/staging
terragrunt run-all init
terragrunt run-all plan
terragrunt run-all apply
```

> alternatively you can also choose to execute only one module (in this case it’s the same as above since there’s only one module anyway):

Reviewer: ```suggestion
Alternatively you can also choose to execute only one module (in this case it’s the same as above since there’s only one module anyway):
```

```
cd terragrunt/env/staging/bucket
terragrunt init
terragrunt plan
terragrunt apply
```

> if you then wanted to simulate the production environment, it would be the same commands as above, but you’d start in `terragrunt/env/production` and run the same commands

Reviewer: ```suggestion
If you then want to simulate the production environment, it would be the same commands as above, but you start in `terragrunt/env/production` and run the same commands
```

Author: @patheard I can't merge this PR just in case you are expectiing I should do so
